suite-common/wallet-graph/src/graphReducer.ts

```diff
@@ -49,10 +51,24 @@ const updateSectionPoints = (
 };
 
 export const prepareGraphReducer = createReducerWithExtraDeps(graphInitialState, builder => {
-    builder.addCase(getGraphPointsForAccountsThunk.fulfilled, (state, action) => {
-        updateSectionPoints(state, action.payload);
-    });
+    builder
+        .addCase(getAllAccountsGraphPointsThunk.fulfilled, (state, action) => {
+            if (action.payload) {
+                updateSectionPoints(state, action.payload);
+            }
+        })
```

Comment:
Please type payloads here and other actions as well. Or maybe I don't understand why payload could be undefined?

```suggestion
        .addCase(getAllAccountsGraphPointsThunk.fulfilled, (state, { payload }: <TYPE_HERE>) => {
                updateSectionPoints(state, payload);
        })
```

Reply:
> I don't understand why payload could be undefined?

Both thunks in `graphThunks.ts` can be fulfilled but return undefined (for `getSingleAccountGraphPointsThunk` it is when no account is found by requested accountKey .... for `getAllAccountsGraphPointsThunk` when no accounts exist for requested network symbols...)

Reply:
Maybe will be better throw instead of return in that case. Account not found is valid reason for that.

Reply:
11e4de766fb57bfccbb6ea85dd9d8e4063d9703a